build.gradle

```diff
@@ -21,12 +21,14 @@ repositories {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```

Comment:
This dependecy is already added on line 28

Reply:
ok, deleted.